templates/netlify/remix.init/index.js

```diff
@@ -59,9 +58,6 @@ async function updatePackageJsonForEdge(directory) {
     scripts: {
       ...scripts,
       dev: scripts["start"],
-      build: "run-s build:*",
```

Comment:
Should it still have the `build` command?

Reply:
It's not necessary as I was overriding the existing one which is `build: "build remix"`. See https://github.com/nickytonline/remix/blob/c419ae8b384e02eba691566880f5fe1bedaa0508/templates/netlify/package.json#L5